cli/script/command-executor.ts

```diff
@@ -623,7 +653,7 @@ function logout(command: cli.ICommand): Promise<void> {
 function formatDate(unixOffset: number): string {
     var date: moment.Moment = moment(unixOffset);
     var now: moment.Moment = moment();
-    if (now.diff(date, "days") < 30) {
+    if (Math.abs(now.diff(date, "days")) < 30) {
```

Comment:
are you sure this is correct? [`fromNow`](http://momentjs.com/docs/#/displaying/fromnow/) by the looks of the docs, is meant for dates in the past. if you wanted to support dates in the future, the correct method to call would be [`toNow()`](http://momentjs.com/docs/#/displaying/tonow/).

```
var dateDiff = now.diff(date, "days");
if (dateDiff >= 0 && dateDiff < 30) {
    return date.fromNow();
} else if (dateDiff > -30) {
    return date.toNow();
}
```